tests/unit/cli/test_errors.py

```diff
@@ -111,11 +111,22 @@ def test_handler_raven_but_no_sentry_feature_flag(self):
 
         self.assert_exception_traceback_exit_1_with_debug()
 
-    @mock.patch('snapcraft.cli._errors.RavenClient')
-    @mock.patch('snapcraft.cli._errors.RequestsHTTPTransport')
     @mock.patch('click.confirm', return_value=True)
     def test_handler_traceback_send_traceback_to_sentry(
-            self, click_confirm_mock, raven_request_mock, raven_client_mock):
+            self, click_confirm_mock):
+        try:
+            import raven  # noqa: F401
+        except ImportError:
+            self.skipTest('raven needs to be installed for this test.')
```

Comment:
I assume we're doing this as opposed to simply installing Raven because it's not available in xenial?

Reply:
correctissimo!